Assets/Mirror/Runtime/Transport/MultiplexTransport.cs

```diff
@@ -138,7 +132,7 @@ int ToTransportId(int connectionId)
             return connectionId % transports.Length;
         }
 
-        void InitServer()
+        void AddServerCallbacks()
```

Comment:
separate pr

Reply:
InitServer is removed because it we can't add handler in awake. 

The new method is mostly, but is called in a different place and has a different purpose (before: add all Listener in awake, now: give callbacks to servers before they start)